### PR TITLE
Add translation tests

### PR DESCRIPTION
**Pull request by StefanMavrodiev**
The tests includes:

- Check if keys are ordered alphabetically
- Check if there are duplicate keys
- Check if there are duplicate values

During the development, the current translations are "styled" and fixed.

### CODE REVIEW

File: tests/phpunit/test/Translations/TranslationsTest.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            $sorted = $trans['translations'];
+            ksort($sorted);
+            $this->assertSame($sorted, $trans['translations'], "The file '" . $file . "' is not sorted alphabetically");
+        }
+    }
+
+    /**
+     * Test if there is duplicate values, that not in $duplicates array
+     */
+    public function testDuplicateValues(): void
```
Comment (angelk) [inline tests/phpunit/test/Translations/TranslationsTest.php]: Why do we need such check?

Comment (StefanMavrodiev) [reply tests/phpunit/test/Translations/TranslationsTest.php]: Duplicate translations means duplicate key values, right?
**Home** -> **Начало**
**home** -> **Начало**
**Home** -> **Начало**

Comment (angelk) [reply tests/phpunit/test/Translations/TranslationsTest.php]: yes.
The IDE should detect such issues, code reviews too.
Having the translation alphabetically sorted should make duplicate keys obvious.
We may also change to YAML format for translations. YAML do not allow duplicate keys.

File: tests/phpunit/test/Translations/TranslationsTest.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            $sorted = $trans['translations'];
+            ksort($sorted);
+            $this->assertSame($sorted, $trans['translations'], "The file '" . $file . "' is not sorted alphabetically");
+        }
+    }
+
+    /**
+     * Test for duplicate keys
+     */
+    public function testDuplicateKeys(): void
```
Comment (angelk) [inline tests/phpunit/test/Translations/TranslationsTest.php]: The order test should be enough.
We can remove this `testDuplicateKeys` keys and merge the changes

Comment (StefanMavrodiev) [reply tests/phpunit/test/Translations/TranslationsTest.php]: Sure

**Comment by StefanMavrodiev** [comment]
testDuplicateKeys removal was not pushed. #161

**Pull request merged**